### PR TITLE
Lxc to docker

### PR DESCRIPTION
**Pull request by roxell**

### CODE REVIEW

File: projects/lt-qcom/fastboot.jinja2 [hunk 1]
```diff
@@ -84,10 +84,10 @@
     namespace: target
     images:
       boot:
-        url: lxc:///{{LXC_BOOT_FILE}}
+        url: lxc:///{{DOCKER_BOOT_FILE}}
```
Comment (mwasilew) [inline projects/lt-qcom/fastboot.jinja2]: so qcom will still use lxc?

File: submit_for_testing.py [hunk 2]
```diff
@@ -398,7 +398,7 @@ def main():
             logger.debug(testpath)
             logger.debug(test)
             container = client.containers.run(
-                image="lavasoftware/amd64-lava-server:2019.12",
+                image="lavasoftware/amd64-lava-server:2020.07",
```
Comment (mwasilew) [inline submit_for_testing.py]: maybe we should use 'latest' here?

Comment (roxell) [reply submit_for_testing.py]: isn't it better to update this when we see/find issues with the current one we have?

Comment (mwasilew) [reply submit_for_testing.py]: It's only used for checking the generated jobs, so latest should be the target. If you want to test against actual instance, it should be done differently

Comment (roxell) [reply submit_for_testing.py]: OK I'll update that

**Pull request merged**